docs/madr/decisions/037-configure-all-gateways.md

```diff
@@ -0,0 +1,78 @@
+# Configuring many gateways
+
+- Status: accepted
+
+## Context and Problem Statement
+
+This document aims to clarify how to configure all gateways or a subset of them, especially when adjustments, such as in `MeshTimeout`, are needed compared to regular Mesh traffic.
+
+## Decision Drivers
+
+- When we set up default rules, we want to use different settings for gateways than we do for regular Mesh applications.
+
+## Considered Options
+
+- Introduce new kind `MeshGatewaysSubset`
+- Add label to all gateways and target them by kind `MeshSubset`
+
+## Decision Outcome
+
+- ?
+
+## Introduce new kind `MeshGatewaysSubset`
```

Comment:
The naming of this kind is a bit confusing but hard to find other name
- MeshGatewaysSubset - Subset of all gateways
- MeshGateway - specific gateway by name
- MeshGateways - all gateways, but it's too similar to MeshGateway
- others?

Another option could be to introduce an additional field in a `targetRef` object but it just moves functionality of kind outside so I don't think this is a good option.

Reply:
MeshGatewaySuperset :) i.e. if we want to select all MeshServices we can add `MeshServiceSuperset`

Reply:
Hmm true, or actually just `MeshGatewaySet` right?

Reply:
TBH I'd prefer `...Superset`, there are definitions in Wiki on "superset/subset" and it seems more clear we're trying to oppose "superset" to "subset". While if you're reading `...Set` it doesn't tell you much on its own